consensus/istanbul/validator/weighted.go

```diff
@@ -813,7 +818,7 @@ func calcTotalAmount(weightedValidators []*weightedValidator, stakingInfo *rewar
 		}
 	}
 
-	logger.Debug("calculate totalStaking", "UseGini", stakingInfo.UseGini, "Gini", stakingInfo.Gini, "totalStaking", totalStaking, "stakingAmounts", stakingAmounts)
+	logger.Debug("calculate totalStaking", "UseGini", stakingInfo.UseGini, "Gini", gini, "totalStaking", totalStaking, "stakingAmounts", stakingAmounts)
```

Comment:
Doesn't this `gini` always print `-1`?

Reply:
Fixed in https://github.com/klaytn/klaytn/commit/4992fab050621b1ff3261c4d810e291b521c83c7.